README.md

```diff
@@ -1 +1,47 @@
-# display-input-data
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# HTML Forms
+
+> This project consists of building an HTML document that matches the appearance of mint.com’s signup page.
+
+![screenshot][screenshot_url]
+
+[screenshot_url]: https://github.com/jrai0792/display-input-data/blob/main/127.0.0.1_60302_(CC's%20Laptop).png?raw=true
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML,
+- CSS
+
+
```

Comment:
- [ ] **[OPTIONAL]** It would be nice to have a live demo link so anyone that sees your page can look at your actual page, for this, you can use https://raw.githack.com and put the link to your index.html file